tests/config/install-prettier.js

```diff
@@ -14,15 +14,15 @@ if (!allowedClients.has(client)) {
 
 module.exports = (packageDir) => {
   const tmpDir = tempy.directory();
-  const fileName = execa
-    .sync("npm", ["pack"], { cwd: packageDir })
-    .stdout.trim();
+  const fileName = execaSync("npm", ["pack"], {
```

Comment:
Did you try use it directly? This file should be able to use ESM.

Reply:
Never mind, I forgot we need run tests on Node.js 10

Reply:
I've noticed Node.js 10 doesn't support `"imports"` in `package.json`. I'll consider the other way.